modules/client-configuration/pages/autoinst-pxeboot.adoc

```diff
@@ -0,0 +1,99 @@
+
+In the installation menu, the first of the choices is [guimenu]``Local boot``. Selecting this choice continues the boot process from the local disk drives. This option is selected automatically if no distribution was selected after a certain time. This is a security measure to prevent from starting an autoinstallation if no human operator is there to select one of the distributions.
+
+Alternatively, the installation might start automatically from one of the distributions without manual intervention. This is called "unattended provisioning".
+
+The "bare metal" functionality is a variant of unattended provisioning where the bootloader file only registers the client at the {productname} Server and does not start the installation. You may then trigger a reinstallation as described in previous section xref:client-configuration:autoinst-reinstall.adoc[Reinstalling Registered Systems].
+
+.Procedure: Performing an Installation with PXE Boot
+. Prepare the DHCP server, see xref:client-configuration:autoinst-pxeboot.adoc#prepare-the-dhcp-server[Prepare the DHCP Server].
+. Prepare an autoinstallation distribution, see xref:client-configuration:autoinst-distributions.adoc[Autoinstallation Distributions].
+. Prepare an autoinstallation profile, see xref:client-configuration:autoinst-profiles.adoc[Autoinstallation Profiles].
+. Reboot the client, and chose a distribution to install if needed.
+
+Some other steps are optional:
+
+* if you want to use a proxy as the installation server, see xref:client-configuration:autoinst-pxeboot.adoc#synchronize-the-tftp-tree-with-proxies[Synchronize the TFTP Tree with Proxies];
+* if you desire a fully unattended provisioning, see xref:client-configuration:autoinst-xxxxx.adoc[].
+
+
+[[prepare-the-dhcp-server]]
+== Prepare the DHCP Server
+
+The PXE boot process uses DHCP to find the TFTP server. The {productname} Server or its proxies can act as such a TFTP server.
+
+You must have administrative access to the network’s DHCP server. Edit the DHCP configuration file so that it points to the {productname} Server as the TFTP boot server:
+
+.Procedure: Example for Configuring the ISC DHCP Server
+. On the DHCP server, as root, open the [path]``/etc/dhcpd.conf`` file.
+. Modify the declaration for your client following this model:
+----
+  host myclient { (...)
+                  next-server 192.168.2.1;
+                  filename "pxelinux.0"; }
+----
+. Save the file and restart the [systemitem]``dhcpd`` service.
+
+This example:
+
+* directs PXE client "myclient" to the installation server at 192.168.2.1.
+* makes it retrieve the pxelinux.0 bootloader file.
+
+Alternatively, if your DHCP server is registered at {productname}, you can configure it using the DHCPd formula instead:
+
+.Procedure: Example for configuring ISC DHCP Server Using DHCPd Formula
+. Configure the DHCPd formula.
```

Comment:
Probably better to use "navigate to the DHCPd formula" here, rather than sending people off to look it up.

Reply:
No, it is a full and complicated process of its own.

To avoid "sending" the people there, we would need to bring much more details here.

Reply:
no, as it's an integrated part of the procedure

maybe we can reproduce here some of the information that is there...

but i don't understand - there are other long procedures here and there where you give all the details somewhere else

Reply:
Surely the simplest thing to do here is just to make the first step "Navigate to Systems[System List], select the client you want to change, and go to the [Formulas] tab to enable the DHCPd formula"? Then the following steps are the specific settings for the formula, and it can end with "For more information about the DHCPd formula, see [salt guide]"? It's not about the length of the procedure, just trying to make sure we're sticking to conventions, making it easy for the reader to follow, and not replicating information if we don't need to.

Reply:
Sounds clean, trying to do that.